Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -1347,24 +1389,27 @@ theorem toL1_sub (f g : α → β) (hf : Integrable f μ) (hg : Integrable g μ)
 #align measure_theory.integrable.to_L1_sub MeasureTheory.Integrable.toL1_sub
 
 theorem norm_toL1 (f : α → β) (hf : Integrable f μ) :
-    ‖hf.toL1 f‖ = ENNReal.toReal (∫⁻ a, edist (f a) 0 ∂μ) := by simp [to_L1, snorm, snorm'];
+    ‖hf.toL1 f‖ = ENNReal.toReal (∫⁻ a, edist (f a) 0 ∂μ) := by
+  simp [toL1, snorm, snorm']
   simp [edist_eq_coe_nnnorm]
```

Comment:
Can these just be combined?

Reply:
These can't because `snorm` must be used earlier than `edist_eq_coe_nnnorm`.